### ISSUE TITLE
Consider daylight time when non-daylight timezone is requested during summer time

### ISSUE THREAD

**Issue by gsauthof** [dommmel/mytime.io#13]
For example, when requesting

    https://mytime.io/11am/PST

I get:

```
11am PST is
9:00:00 PM
your time
assuming the date is today and your time zone is [CEST](https://www.google.com/search?q=%22CEST%22+timezone+site:wikipedia.org&btnI)
```

which is correct at a level, but usually not that helpful.

I mean at the one hand the page assumes CEST based on what the browser provides and on the other hand you assume that the destination timezone doesn't observe daylight time anymore.

It's kind of common for humans to communicate timezones in appointments imprecisely.

For example, to continue to use CET and PST in the summer when daylight time is active, and not CEST, PDT etc. because people don't care enough.

Thus, it would be great for your service to provide some extra logic, e.g. cross-check whether actually daylight time is active, and then print an additional hint about that, i.e. also show the computed source time under a human interpretation.

**Comment by vcunat** [comment]
Yes, it's a bit confusing to print timezone as CEST when it's really CET.  They are one hour apart.
